diff -urN protux-0.19.3.orig/src/Song.cc protux-0.19.3/src/Song.cc
--- protux-0.19.3.orig/src/Song.cc	2003-08-16 01:34:54.000000000 +0200
+++ protux-0.19.3/src/Song.cc	2003-09-10 00:18:40.000000000 +0200
@@ -2334,7 +2334,7 @@
                         }
                 else if (actionName == "WORK ON NEXT EDGE")
                         {
-                        long long w = 999999999999; // almost 1 trillion
+                        long long w = 2147483647;
                         for (int i=0; i<numTracks; i++)
                                 {
                                 AudioClip* c=track[i]->get_clip_after(workingBlock);
